tests/cmd/stability/main.go

```diff
@@ -191,12 +191,16 @@ func run() {
 			cluster.TiKVPreStartScript = strconv.Quote("")
 			cluster.TiDBPreStartScript = strconv.Quote("")
```

Comment:
do we need to add "# noop" to `TiDBPreStartScript` in this PR?

Reply:
I prefer adding it